Documentation.md

```diff
@@ -164,590 +174,606 @@ and made available to your JS client code.
+* `routingProfile`: See `agent.getRoutingProfile()` for more info.
+* `softphoneAutoAccept`: Indicates whether auto accept soft phone calls is enabled.
+* `softphoneEnabled`: See `agent.isSoftphoneEnabled()` for more info.
 * `username`: The username for the agent as entered in their Amazon Connect user account.
 
 ### `agent.getAgentStates()`
-```
+```js
 var agentStates = agent.getAgentStates();
 ```
-Gets the list of selectable `AgentState` API objects.  These are the agent states that can be
-selected when the agent is not handling a live contact.  Each `AgentState` object contains
+Gets the list of selectable `AgentState` API objects. These are the agent states that can be
+selected when the agent is not handling a live contact. Each `AgentState` object contains
 the following fields:
 
-* `type`: The `AgentStateType` associated with this agent state.
+* `agentStateARN` The agent state ARN.
+* `type`: The agent state type represented as a `AgentStateType` enum value.
 * `name`: The name of the agent state to be displayed in the UI.
 
 ### `agent.getRoutingProfile()`
-```
+```js
 var routingProfile = agent.getRoutingProfile();
 ```
-Gets the agent's routing profile.  The routing profile contains the following fields:
+Gets the agent's routing profile. The routing profile contains the following fields:
 
+* `channelConcurrencyMap`: See `agent.getChannelConcurrency()` for more info.
+* `defaultOutboundQueue`: The default queue which should be associated with outbound contacts. See `queues` for details on properties.
 * `name`: The name of the routing profile.
-* `queues`: The queues contained in the routing profile.
-* `defaultOutboundQueue`: The default queue which should be associated with outbound contacts.
+* `queues`: The queues contained in the routing profile. Each queue object has the following properties:
+  * `name`: The name of the queue.
+  * `queueARN`: The ARN of the queue.
+  * `queueId`: Alias for `queueARN`.
+* `routingProfileARN`: The routing profile ARN.
+* `routingProfileId`: Alias for `routingProfileARN`.
 
-### `agent.getChannelConcurrency`
-```
-if (agent.getChannelConcurrency("VOICE")) { ... }
-```
-OR
-```
-const concurrencyMap = agent.getChannelConcurrency();
+### `agent.getChannelConcurrency()`
+```js
+var concurrencyMap = agent.getChannelConcurrency(); // e.g. { VOICE: 0, CHAT: 2 }
+
+// OR
+
+if (agent.getChannelConcurrency("CHAT") > 1) { /* ... */ }
 ```
-Gets either a boolean represented by a 1 or 0, or a map of channel type to 1 or 0. 1 represents an enabled channel. 0 represents a disabled channel.
-* `channel`: A string ENUM representing the channel whose value to return. 
-The ENUM options are currently `"VOICE"` and `"CHAT"`
+Gets either a number or a map of `ChannelType`-to-number indicating how many concurrent contacts can an agent have on a given channel. 0 represents a disabled channel. The optional `channel` argument must be a value of the enum `ChannelType`.
```

Comment:
Updated this documentation to reflect that concurrency can be higher than 1 (e.g. chat).